velox/expression/SignatureBinder.h

```diff
@@ -40,6 +34,29 @@ class SignatureBinder {
     }
   }
 
+  // Return true if actualType can bind to typeSignature and update bindings_
```

Comment:
Use ///

Reply:
Sounds good. Shall we always use /// (as opposed to //) for comments above classes or functions? Or do we follow some other rules?

Reply:
Here is what the coding style says:

https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md

```
For comments on headers for classes, functions, methods and member variables (the ones to be exposed as documentation), use /// (three slashes) at the front of each line. 
```